tests/hello2.c

```diff
@@ -0,0 +1,9 @@
+#include <stdio.h>
```

Comment:
Why do we need another Hello test program?

Reply:
The hello2.c is to check the function parameters work correctly or not.

Reply:
Prepare new test cases which emphasize on parameter checks. The naming should be explicit.